### PR TITLE
Shell smashing - New game at Hollow Eve.

### PR DESCRIPTION
**Pull request by Dartellum**
Just going to reuse the pumpkin_junk list.

### CODE REVIEW

File: smash-shells.lic [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    DRC.fix_standing
+    case DRC.bput('grab box', 'roundtime', 'You are too injured', 'Perhaps that would work better', 'when you have enough coins', 'You should clean your hands')
+    when 'You are too injured', 'You should clean your hands'
+      DRCT.walk_to(16_150)
+      exit
+    when 'when you have enough coins'
+      DRCT.walk_to(16_315)
+      case DRC.bput('withdraw 3 platinum', 'we are not lending money at this time', 'The clerk counts out')
+      when 'we are not lending money at this time'
+        exit
```
Comment (rpherbig) [inline smash-shells.lic:26]: 💡 Good place to tell the user they need to get more money

File: smash-shells.lic [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    end
+    pause 1
+    waitrt?
+  end
+
+  DRCT.walk_to(16_237)
+
+  case DRC.bput('break my shell', 'The shell', 'Break what', 'need a free hand', 'You should hold')
+  when 'need a free hand', 'Break what'
+    DRCI.stow_hands
```
Comment (rpherbig) [inline smash-shells.lic:38]: ❓ Do you need to retry 'break my shell; after emptying hands?

File: smash-shells.lic [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  pause
+
+  while thing = DRC.left_hand || DRC.right_hand
+    case thing
+    when *junk_list
+      DRC.bput("put #{thing} in bucket", 'You drop')
+    else
+      case DRC.bput("stow #{thing}", 'You put', 'to fit in the')
+      when 'to fit in the'
+        exit
```
Comment (rpherbig) [inline smash-shells.lic:49]: 💡 Another good place to explain to the user

File: smash-shells.lic (hunk 1 above)
Comment (Dartellum) [reply smash-shells.lic:26]: Added message

File: smash-shells.lic (hunk 3 above)
Comment (Dartellum) [reply smash-shells.lic:49]: Added message.

File: smash-shells.lic (hunk 2 above)
Comment (Dartellum) [reply smash-shells.lic:38]: Need more testing. I took the pumpkin script and changed it to shells.

**Comment by rpherbig** [comment]
Looks good to me ✔️ 

> Need more testing

Let me know when this is ready to be merged

File: smash-shells.lic (hunk 2 above)
Comment (Dartellum) [reply smash-shells.lic:40]: No, as it will loop through again until injured.

**Comment by Dartellum** [comment]
> Let me know when this is ready to be merged

The script is ready for merging.

**Pull request merged**